projects.md

```diff
@@ -113,25 +92,46 @@ https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrat
 
 # Projects
 
-Here we highlight some (not all) Bitcoin projects we care about, for a few reasons:
+We don’t just chat about design and create resources for designers, we are also deeply involved with projects to power up Bitcoiners through good design. If you are interested in helping out, check the links for each project, ask directly in the communities of those projects, or reach out in our [Slack]({{ site.slack_invite_url }}).
```

Comment:
Not a big fan of the copy. However, I am lacking coffee so can't make a sensible suggestion either.
What the paragraph should say is:
- Why do we collaborate with the projects
- How does the collaboration happen
- How can projects get involved 
- How can community members help out

Reply:
I revised the copy, let me know what you think now.

Reply:
Way better, I've suggested minor simplification, let me know if you're okay with the suggestion [here](https://github.com/BitcoinDesign/Guide/pull/179#discussion_r594960698)